test/remote/gateways/remote_usa_epay_transaction_test.rb

```diff
@@ -22,6 +22,20 @@ def test_successful_purchase_with_track_data
     assert_success response
   end
 
+  def test_successful_authorization_with_manual_entry
+    @credit_card.manual_entry = true
```

Comment:
I'd lean towards using the `@creditcard` variable for this, but if you'd rather rename it I'm cool with that too.